nebula-logger/extra-tests/tests/LoggerBenchmarking_Tests.cls

```diff
@@ -0,0 +1,40 @@
+@SuppressWarnings('PMD.MethodNamingConventions')
+@IsTest
+private class LoggerBenchmarking_Tests {
+    private static final LoggerParameter__mdt TRANSACTION_LIMIT_PARAM = new LoggerParameter__mdt(DeveloperName = 'StoreTransactionLimits');
+    private static final LoggerParameter__mdt ORG_LIMIT_PARAM = new LoggerParameter__mdt(DeveloperName = 'StoreOrganizationLimits');
+
+    @IsTest
+    static void it_benchmarks_while_setting_transaction_limits() {
+        TRANSACTION_LIMIT_PARAM.Value__c = JSON.serialize(true);
+        ORG_LIMIT_PARAM.Value__c = TRANSACTION_LIMIT_PARAM.Value__c;
+
+        logForEntries(500);
+
+        System.Assert.isTrue(true, 'Made it here without an uncatchable System.LimitException being thrown');
+    }
+
+    @IsTest
+    static void it_benchmarks_without_setting_limit_info() {
```

Comment:
We don't have to add it in this PR, but it could be neat to add one more test method that has `StoreApexHeapSizeLimit` set to `false`, and `StoreTransactionLimits` set to `true`.